### PR TITLE
Mixxx 2.3.3 Release Candidate

### PR DESCRIPTION
**Pull request by daschuer**
This is a 2.3.3 Release Candidate.

### CODE REVIEW

File: packaging/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+mixxx (2.3.2-1~bionic) bionic; urgency=medium
+
+  * Build of 2.3.2
+
+ -- RJ Skerry-Ryan <rryan@mixxx.org>  Sun, 30 Jan 2022 20:37:54 +0000
+
 mixxx (2.3.1-1~bionic) bionic; urgency=medium
 
   * Build of 2.3.1
```
Comment (Swiftb0y) [inline packaging/debian/changelog:9]: whats going on with the version offset here?

Comment (daschuer) [reply packaging/debian/changelog:9]: The debian tools automatically add the change log entry for the current build version. 
I have copied over just the auto-generated log entry form the last release. Else we would have a gap after the future release.

Comment (Swiftb0y) [reply packaging/debian/changelog:9]: thank you for the explanation.

**Comment by ronso0** [comment]
Is there a chance #4793 can still slip in? This would complement the recent passthrough fixes.

**Comment by daschuer** [comment]
Yes of cause.

**Comment by daschuer** [comment]
Once this is merged, I will all out for Smoke-Tests of the created release candidates and then proceed to the release. 
OK?

**Comment by daschuer** [comment]
In the meantime lets prepare a release blog post.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 * Pioneer-DDJ-SB3: Fix controller breaking when releasing the shift button [#4659](https://github.com/mixxxdj/mixxx/pull/4659)
 * Traktor S3: Push two deck switches to explicitly clone decks [#4665](https://github.com/mixxxdj/mixxx/pull/4665) [#4671](https://github.com/mixxxdj/mixxx/pull/4671) [lp:1960680](https://bugs.launchpad.net/mixxx/+bug/1960680)
-* Behringer DDM4000: Improve stability and add soft-takeover for encoder knobs [#4318](https://github.com/mixxxdj/mixxx/pull/4318)
+* Behringer DDM4000: Improve stability and add soft-takeover for encoder knobs [#4318](https://github.com/mixxxdj/mixxx/pull/4318) [#4799](https://github.com/mixxxdj/mixxx/pull/4799)
```
Comment (Swiftb0y) [inline CHANGELOG.md:7]: Sorry for introducing this last minute. I forgot we kinda froze 2.3 until the patch release is out.

**Review by Swiftb0y — APPROVED** [review]
LGTM. What now? Merge?

**Comment by Holzhaus** [comment]
> LGTM. What now? Merge?

Before tagging, let's first write the blogpost and review it until it's ready to merge. Otherwise we are in a situation where distros begin to pick up the new mixxx release even though we didn't announce it yet.

**Comment by daschuer** [comment]
Yes, so we have these steps:
* Merge this 
* wait until we have builds 
* Ask testers for some final checks
* Write a blog post 
* Tag the build and create the release binaries
* Merge the blog. 

@Holzhaus do you have interest to start the blog post?

**Comment by Swiftb0y** [comment]
I'll take care of the blogpost.

**Comment by Swiftb0y** [comment]
Blog post is ready for review: mixxxdj/website/pull/272

I'd suggest to merge this PR now, then I'll add the changelog to the blog entry.

**Comment by daschuer** [comment]
Yes, go ahead.

**Pull request merged**